app/components/service/includes/container-specs.html

```diff
@@ -1,4 +1,4 @@
-<div>
+<div id="service-resources">
```

Comment:
The `service-resources` id should be placed on the service specification panel located in `app/components/service/includes/resources.html` instead of `container-specs.html`.